### DISCUSSION TITLE
Grupo para el proximo Evento

### DISCUSSION THREAD

**Discussion by AleBT75** [Launch-X-Latam/MisionFrontEnd#524] [Team Posts]
Buen dia desde donde lo lean, estoy en busca de un grupo, como muchos aqui sigo aprendiendo y aun no tengo todo claro asi que tomo clases aparte de esta experiencia para estar preparado, si alguien gusta hacer un grupo por favor responda este mensaje, espero a alguien le interese

**Comment by AlmaBarreraP** [comment]
Hola! Puedo unirme a tu equipo? Mi correo es:  lx03477@innovaccion.mx

**Comment by Ayuya2001** [comment]
Hola podría unirme aquí?
Mi correo es: ***@***.*** <http://innovaccion.mx/>*

**Comment by DavisonH** [comment]
Tenemos dos cupos por llenar el que este interesado, correo: lx12963@innovaccion.mx

**Comment by Jpulido6** [comment]
Adoptame bro, ando sin grupo🙏🏻

**Comment by Tfrnk** [comment]
Bro tambien ando sin grupo, me podrias agregar por favor

**Comment by Braddycruz1** [comment]
Yo quiero estar en el grupo 😔

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by AleBT75** [comment]
Quienes gusten déjenme su correo o escríbanme aquí lx12185@innovaccion.mx con gusto les respondo :D

**Comment by DiegoIrribarren** [comment]
Hola busco grupo tmb

Diego V. Irribarren

**Comment by AleBT75** [comment]
Escribeme al correo o directamente a Microsoft Teams si gustas para mejor comunicación :)

**Comment by aramirex** [comment]
El El mar, 8 de noviembre de 2022 a la(s) 22:56, Diego Irribarren <
***@***.***> escribió:

> Hola busco grupo tmb
>
> Diego V. Irribarren
>
> On Tue, Nov 8, 2022, 22:24 AlmaBarreraP ***@***.***> wrote:
>
> > Hola! Puedo unirme a tu equipo? Mi correo es: ***@***.***
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/orgs/Launch-X-Latam/teams/mision-backend/discussions/54/comments/1
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/A3IT2SQ4KVIKXNMHCUSHYEDWHMKQRANCNFSM6AAAAAAR26VIOM
> >
> > .
> > You are receiving this because you are subscribed to this thread.Message
> > ID: ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/Launch-X-Latam/teams/mision-backend/discussions/54/comments/8>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3QFXIK77YSX2IGCOC62VX3WHMVHNANCNFSM6AAAAAAR26VIOM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
Si puedes agregarme al grupo por favor, mi correo es: ***@***.***

**Comment by Sacoma1** [comment]
Hola ale, me llamo miguel , estoy en busca de equipo lx16970@innovaccion.mx, espero aun tengaas cupo

**Comment by AleBT75** [comment]
Buenas noches Miguel, ahora mismo te escribo vía Microsoft Teams 😁
